cmd/state/internal/cmdtree/packages.go

```diff
@@ -67,6 +68,11 @@ func newInstallCommand(prime *primer.Values) *captain.Command {
 				Description: locale.T("package_flag_ts_description"),
 				Value:       &params.Timestamp,
 			},
+			{
+				Name:        "force",
+				Description: locale.Tl("package_flag_force_description", "Ignore security policy preventing packages with CVEs from being installed (not recommended)"),
+				Value:       &force,
+			},
```

Comment:
We should not be adding force as a local flag. It should be a global flag just like non-interactive. Whether it is relevant to use on a command and communicating that through the UI is not a requirement, just like with non-interactive.